index.html

```diff
@@ -1,198 +1,234 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Sameh's portfolio</title>
 </head>
+
 <body class="font">
-<section class="page1">
-    <header class="topbar">
-        <a>Welcome</a>
-        <nav class="menu"><img src="./images/icons/Union.png" alt="menu"></nav>
-        <div class="desktoplinks">
-            <a>Portofolio</a>
-            <a>About</a>
-            <a>Contact</a>
+    <section class="page1" id="page1">
+        <header class="topbar">
+            <nav class="navbar">
+                <a class="navbar-brand" href="#page1">Welcome</a>
+                <div class="burger" id="burger">
+                 <div class="hold">
+                  <div class="dash"></div>
+                  <div class="dash"></div>
+                  <div class="dash"></div>
+                </div>
+            </div>
+                <div class="menu-block" id="menu-block">
+                    <img id="close" class="close" src="./images/icons/Iconx.png" alt="close-btn">
+                    <ul class="menu-items" id="menu-items">
+                    <li><a id="menuitem1" href="#works">Portfolio</a></li>
+                    <li><a id="menuitem2" href="#about">About</a></li>
+                    <li><a id="menuitem3" href="#contact">Contact</a></li></ul>
```

Comment:
- [ ] [OPTIONAL] it's better to use ```class``` to style elements instead of ```id```, so you follow the HTML best practices